ssh/tailssh/accept_env_test.go

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package tailssh
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+func TestMatchAcceptEnvPattern(t *testing.T) {
+	testCases := []struct {
+		pattern string
+		target  string
+		match   bool
+	}{
+		// Positive cases
```

Comment:
Nit - it might be easier to follow the tests if positive and negative test cases for the same pattern were grouped together, as opposed to grouping all positive test cases together.

Reply:
Oh fair, like have them grouped like:

```go
{pattern: "TEST_*", target: "TEST_A_LONG_ENVIRONMENT_VARIABLE_NAME", match: true},
{pattern: "TEST_*", target: "TEST", match: false},

{pattern: "??", target: "CC", match: true},
{pattern: "??", target: "123", match: false},
```

you mean?